scripts/generate-app-config/generatePagesConfig.js

```diff
@@ -136,6 +136,104 @@ const buildPageConfig = (filePaths, generatePagesOptions, namespace) => (
   }, {})
 );
 
+const getScreenshotPatterns = generatePagesOptions => (
```

Comment:
Could we break this screenshot code out into it's own file? It seems isolated enough and I would like to keep filesize down.